src/cms/api/protected-area/models/protected-area.settings.json

```diff
@@ -181,6 +181,11 @@
     "parkSubPages": {
       "via": "protectedArea",
       "collection": "park-sub-page"
+    },
+    "seo": {
+      "type": "component",
+      "repeatable": false,
+      "component": "parks.seo"
     }
```

Comment:
I think you'd need to add the same setting to `site.setting.json` to use `seo` component in `site.js`

Reply:
I'm using the component in the front-end, but not using any of the fields in site pages. For now it's just using the park and site name for the title, and truncating the description for the meta description.

We should integrated the SEO component in eventually, but not for MVP.